local/deprecatedFlagsSupport.json

```diff
@@ -10,5 +10,11 @@
       "old_name": "whitelisted-subnets",
       "new_name": "track-subnets",
       "value_map": ""
+  },
+  {
+      "version": "v1.10.4",
```

Comment:
we have to adjust this version to caminogo's version ie v1.1.4-rc0. Not sure if we can reference RCs here though

Reply:
CNR will get version from node, which will be `"caminogo: v1.1.4-rc0, commit: 3e2156c73\ncompat: v1.1.5 [database: v1.4.5]\n"`
Then it cuts it down to `"v1.1.4"` and compare it with value from json

Reply:
But jup, thanks, should replace 1.10.4 with 1.1.4